history/management/commands/pull_deposits.py

```diff
@@ -33,6 +32,4 @@ def handle(self, *args, **options):
             d.txid = txid
             d.type = 'deposit' if amount > 0 else 'withdrawal'
             d.status = status
-            d.created_on = created_on
```

Comment:
I think @owocki intention was to save the time of creation on the poloniex server with this. `deposit_created_on`